modules/statistics/statistics.js

```diff
@@ -230,13 +232,31 @@ Statistics.prototype.stopRemoteStats = function() {
 
 /**
  * Initializes the callstats.io API.
- * @param peerConnection {JingleSessionPC} the session object
+ * @param {TraceablePeerConnection} tpc the {@link TraceablePeerConnection}
+ * instance for which CalStats will be started.
+ * @param {string} aliasName an alias name for the local endpoint which will be
+ * used for reporting the new CallStats session.
  */
-Statistics.prototype.startCallStats = function(session) {
+Statistics.prototype.startCallStats = function(tpc, aliasName) {
     if (this.callStatsIntegrationEnabled && !this.callStatsStarted) {
+        /* eslint-disable prefer-template */
+        // The confID is case sensitive!!!
+        const confID
+            = this.options.callStatsConfIDNamespace
```

Comment:
I see that in JitsiConference we try to default it to window.location.hostname, so maybe it should be an error.